man/chpldoc.rst

```diff
@@ -59,6 +59,11 @@ reStructuredText as an intermediate format.
 
     [Don't] generate HTML-based documentation (on by default)
 
+**--project-version <**\ *projectversion*\ **>**
+
+    **chpldoc** generates documentation of version *projectversion*
```

Comment:
I prefer the text you use in docsDriver.cpp

Reply:
Yeah sure

Reply:
Is this good "Specify *projectversion* to set the documentation version"
or shall I proceed with "Sets documentation version to *projectversion*?

Reply:
I like the latter best, the first one is fine but feels more verbose for some reason

Reply:
then I'll keep the second one and update along with the updated docs